jax/interpreters/batching.py

```diff
@@ -463,6 +462,12 @@ def _batch_jaxpr(closed_jaxpr, axis_size, in_batched, instantiate, axis_name,
 
 def batch_jaxpr_axes(closed_jaxpr, axis_size, in_axes, out_axes_dest, axis_name,
                      main_type):
+  return _batch_jaxpr_axes(closed_jaxpr, axis_size, tuple(in_axes),
+                           tuple(out_axes_dest), axis_name, main_type)
```

Comment:
nit: it would be a bit faster to already make in_axes and out_axes_dest above tuples, but lgtm

Reply:
True but I thought I'd err on making things easier for the caller.